routers/api/v2/tokens.go

```diff
@@ -0,0 +1,64 @@
+package v2
+
+import (
+	"github.com/gin-gonic/gin"
+	v2 "github.com/unicsmcr/hs_auth/authorization/v2"
+	"github.com/unicsmcr/hs_auth/routers/api/models"
+	"go.uber.org/zap"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func (r *apiV2Router) ServiceToken(ctx *gin.Context) {
+	owner := ctx.PostForm("owner")
+	allowedURIs := ctx.PostForm("allowedURIs")
+	expiresAt := ctx.PostForm("expiresAt")
```

Comment:
Have you seen [ctx.Bind](https://godoc.org/github.com/gin-gonic/gin#Context.Bind)?

You should be able to replace this with:
```
var req struct {
  Owner string `form:"owner"`
  AllowedURIs []UniformResourceIdentifier `form:"allowedURIs"`
  ExpiresAt int64 `form:"expiresAt"`
}
err := ctx.Bind(&req)
```

It would unmarshall the strings into required data types automatically using the JSON unmarshallers. It might be a good idea to do something like this for the `Login` and `Register` methods, I'll create a PR for this change.

With the `req` struct we could also do automatic validation with this library: https://godoc.org/gopkg.in/go-playground/validator.v9 but if you don't have time to take a look at it right now, no problem.